Tasks/google-play-release/task.json

```diff
@@ -104,6 +104,15 @@
             "helpMarkDown": "Checking this box will make the task attempt to update the metadata on your app release"
         },
         {
+            "name": "metadataRootPath",
+            "type": "filePath",
+            "label": "Metadata Root Directory",
+            "defaultValue": "",
```

Comment:
What will happen if I don't specify any root directory? Do you think you may be have "." or something as the default value?